Owncloud iOs Client/Utils/UtilsUrls.m

```diff
@@ -635,4 +635,23 @@ + (NSString *) getFileLocalSystemPathByFileDto:(FileDto *)fileDto andUser:(UserD
     return fullLocalDestiny;
 }
 
+//-----------------------------------
+/// @name getLocalCertificatesPath
+///-----------------------------------
+
+/**
+ * Method used to get only the domain and the protocol (http/https)
```

Comment:
Change "Method used to get only the domain and the protocol (http/https)" and use the correct method comment.
Same thing for getFileLocalSystemPathByFullPath, getFileLocalSystemPathByFullPath